### PR TITLE
Update _press.md

### PR DESCRIPTION
**Pull request by rebeccaradding**
Adds new links #398, removes old Arabic links, adds names of publications.

### CODE REVIEW

File: _harp/about/_press.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-- [Founders & Coders pooling developer talent through free education course](http://www.computerweekly.com/news/2240236082/Founders-Coders-pooling-developer-talent-through-free-education-course)    (Computer Weekly)
-- [How I Became a Web Developer in 5 Months](https://medium.com/learning-new-stuff/from-non-technical-to-hired-in-5-months-d010f601b1bc#.qnysdqs1z) (Medium)
-- [Fusing nonprofits with tech talent](https://medium.com/@wearecast/fusing-nonprofits-with-tech-talent-bddd5ddbe7bb#.16xbm32wb) (Medium)
-- [Founders & Coders: The UK’s Only Free Coding Academy](http://www.intelligenthq.com/innovation-management/founders-coders-the-uks-only-free-coding-academy/)
-- [Web developers crack the tuition code to snap up Young Co-operators Prize](http://www.uk.coop/newsroom/web-developers-crack-tuition-code-snap-young-co-operators-prize)
-- [Founders and Coders Interview](http://uk.farnell.com/founders-and-coders-interview)
+- The Guardian: [Get with the program: the coders offering training for free](https://www.theguardian.com/technology/2015/jul/26/founders-coders-coding-free-training-london) (The Guardian)
```
Comment (sofer) [inline _harp/about/_press.md]: @rebeccaradding We've got a repetition of the source at both the beginning and the end of the link. I marginally prefer `Headline (Source)` to `Source: Headline`

File: _harp/about/_press.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+- Forbes: [The Free Coding Academy Model -- How To Teach Our Next Generation Of Programmers](http://www.forbes.com/sites/adrianbridgwater/2015/08/13/the-free-coding-academy-model-how-to-teach-our-next-generation-of-programmers/#6b07b18829c5) (Forbes)
+- Computer Weekly: [Founders & Coders pooling developer talent through free education course](http://www.computerweekly.com/news/2240236082/Founders-Coders-pooling-developer-talent-through-free-education-course)    (Computer Weekly)
+- Medium: [How I Became a Web Developer in 5 Months](https://medium.com/learning-new-stuff/from-non-technical-to-hired-in-5-months-d010f601b1bc#.qnysdqs1z) (Medium)
+- Medium: [Fusing nonprofits with tech talent](https://medium.com/@wearecast/fusing-nonprofits-with-tech-talent-bddd5ddbe7bb#.16xbm32wb) (Medium)
+- Intelligent HQ: [Founders & Coders: The UK’s Only Free Coding Academy](http://www.intelligenthq.com/innovation-management/founders-coders-the-uks-only-free-coding-academy/)
```
Comment (des-des) [inline _harp/about/_press.md]: This seems broken but might be temporary .. 
image

File: _harp/about/_press.md (hunk 1 above)
Comment (rebeccaradding) [inline _harp/about/_press.md]: Do you want me to change all of them to that format?

**Comment by rebeccaradding** [comment]
@sofer @des-des what needs to be done to get this merged?

**Comment by rebeccaradding** [comment]
@sofer care to merge?

**Pull request merged**